03-specifying-sample-designs.Rmd

```diff
@@ -585,13 +585,13 @@ recs_des <- recs_in %>%
   )
 ```
 
-In specifying the design, the `variables` option was also used to include which variables might be used in analyses. This is optional but can make your object smaller. When printing the design object or looking at the summary, the replicate weight type is re-iterated as `Fay's variance method (rho= 0.5) with 96 replicates and MSE variances`, and the variables are included. No weight or probability summary is included, as was done in some other design objects.
+In specifying the design, the `variables` option was also used to include which variables might be used in analyses. This is optional but can make your object smaller. When printing the design object or looking at the summary, the replicate weight type is re-iterated as `Fay's variance method (rho= 0.5) with 96 replicates and MSE variances`, and the variables are included. No weight or probability summary is included in this output as we have seen in some other design objects.
 
 ### Jackknife method
 
 There are three jackknife estimators implemented in {srvyr} - Jackknife 1 (JK1), Jackknife n (JKn), and Jackknife 2 (JK2). The JK1 method can be used for unstratified designs, and replicates are created by removing one PSU at a time so the number of replicates is the same as the number of PSUs. If there is no clustering, then the PSU is the ultimate sampling unit (e.g., unit). 
 
-The JKn method is used for stratified designs and requires two or more PSUs per stratum. In this case, each replicate is created by deleting one PSU from each stratum, so the number of replicates is the number of total PSUs across all strata. The JK2 method is a special case of JKn when there are exactly 2 PSUs sampled per stratum. For variance estimation, scaling constants must also be specified.
+The JKn method is used for stratified designs and requires two or more PSUs per stratum. In this case, each replicate is created by deleting one PSU from each stratum, so the number of replicates is the number of total PSUs across all strata. The JK2 method is a special case of JKn when there are exactly 2 PSUs sampled per stratum. For variance estimation, scaling constants must also be specified. 
```

Comment:
In the second sentence "In this case..." I'm not sure I'm following what this means.  Does this mean rep1 removes 5 PSUs if there are 5 strata (one from each strata)?  That's what I'm reading it as, but then it could mean that if the stratas have different number of PSUs then some reps are potentially only removing 1 PSU and not 5.  You then reference that the number of replicates is the total number of PSUs across all strata so I think this sentence just needs to be clarified.  I'm really not following it.

Reply:
Last sentence, "For variance estimation..." is the scaling constant always (R-1)/R?  If so let's put this here and not in the syntax section.